packages/nimble-components/src/anchor-tabs/tests/anchor-tabs.stories.ts

```diff
@@ -0,0 +1,132 @@
+        }
+    },
+    // prettier-ignore
+    render: createUserSelectedThemeStory(html`
+        <nimble-anchor-tabs ${ref('tabsRef')} activeid="icons">
+            ${when(x => x.toolbar, html<TabsArgs>`<nimble-tabs-toolbar :innerHTML="${x => x.toolbar}"></nimble-tabs-toolbar>`)}
+            ${repeat(x => x.tabs, html<TabArgs, TabsArgs>`
+                <nimble-anchor-tab
+                    id="${x => x.id}"
+                    ?disabled="${x => x.disabled}"
+                    href="${x => x.href}"
+                    target="iframe"
+                    @click="${(x, c) => c.parent.onTabClick(c.parent.tabsRef, x.id)}"
+                    @keypress="${(x, c) => c.parent.onTabKeypress(c.parent.tabsRef, x.id, c.event)}"
+                >${x => x.label}</nimble-anchor-tab>
+            `)}
+        </nimble-anchor-tabs>
+        <iframe name="iframe" src="${x => x.tabs[0]!.href}"></iframe>
+        <style class="code-hide">
+            nimble-anchor-tabs {
+                padding-bottom: 3px;
+            }
+            iframe {
+                width: 100%;
+                height: 1000px;
+            }
+        </style>
+    `),
+    argTypes: {
+        activeId: {
+            description:
+                "The `id` of the `nimble-anchor-tab` that should be indicated as currently active/selected. It is the application's responsibility to set `activeId` to the tab matching the currently loaded URL."
+        },
+        onTabClick: {
+            table: {
+                disable: true
+            }
+        },
+        onTabKeypress: {
+            table: {
+                disable: true
+            }
+        }
+    },
+    args: {
+        tabs: [
+            {
+                label: 'Icons',
+                id: 'icons',
+                href: '/?path=/story/icons--icons',
+                disabled: false
+            },
+            {
+                label: 'Spinner',
+                id: 'spinner',
+                href: '/?path=/story/spinner--spinner',
+                disabled: false
+            },
+            {
+                label: 'Text Field',
+                id: 'text-field',
+                href: '/?path=/story/text-field--underline-text-field',
+                disabled: false
+            }
+        ],
+        onTabClick: (tabs: AnchorTabs, tabId: string) => {
+            tabs.activeid = tabId;
+            return true;
+        },
+        onTabKeypress: (tabs: AnchorTabs, tabId: string, event: Event) => {
+            switch ((event as KeyboardEvent).key) {
+                case 'Space':
+                case 'Enter':
+                    tabs.activeid = tabId;
+                    break;
+                default:
+            }
+            return true;
+        }
+    }
+};
+
+export default metadata;
+
+export const tabs: StoryObj<TabsArgs> = {};
+
+export const toolbar: StoryObj<TabsArgs> = {
+    args: {
+        toolbar:
```

Comment:
Discussed switching the toolbar to be more like nimble button icons, ie a toggle instead of a separate story and include discussion about slotting content

Reply:
Made it a toggle option. However I didn't add any information about slots because (1) the `nimble-tabs-toolbar` automatically targets the `end` slot (so the user doesn't have to know about it), and (2) we seem to poorly support slotting other content into the `start` and `end` slots in the `nimble-tabs` component. For example, I tried slotting an icon into the tabs' `start` slot, and it comes out poorly aligned:
![image](https://user-images.githubusercontent.com/7282195/213319655-cce8f1b1-936d-4821-983c-dd322d6a4563.png)
Maybe it was intentional that we didn't document these slots?

Reply:
Well, we can't hide the `end` slot, because as I said, the `nimble-tabs-toolbar` depends on slotting itself there. I'll leave the task of hiding the `start` slot to the issue you linked (I'll leave a note in it about this slot).